doc/sailfish-webview-popups-selectpopupinterface.qdoc

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+  \qmltype SelectPopupInterface
```

Comment:
maybe SelectorPopupInterface would be a better name?

Reply:
Thanks, that is clearer; I've changed it.